addons/l10n_hu_edi/tests/test_credit_debit_notes.py

```diff
@@ -0,0 +1,43 @@
+from odoo.tests.common import tagged
+from odoo.addons.l10n_hu_edi.tests.common import L10nHuEdiTestCommon
+from freezegun import freeze_time
+
+
+@tagged('post_install_l10n', '-at_install', 'post_install')
+class TestL10nHuEdiCreditDebitNotes(L10nHuEdiTestCommon):
+    """Tests for Credit and Debit Notes in the Hungarian EDI localization."""
+
+    @classmethod
+    def setUpClass(cls, chart_template_ref='hu'):
+        with freeze_time('2025-01-25T15:28:53Z'):
```

Comment:
We can just use `@freeze_time()` on the methods. It's shorter and easier to see that the date is frozen.

Also, do we need a date that precise ? Can't we just do `2025-01-25` for example ?